runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -108,4 +110,60 @@ default boolean filterLogin()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "chatCountColor",
+		name = "Collapsed Count Color",
+		description = "Configures the color for the number of collapsed messages",
+		position = 8
+	)
+	default Color chatCountColor()
+	{
+		return new Color(0xA00080);
+	}
+
+	@ConfigItem(
+			keyName = "collapseGameChat",
+			name = "Collapse Game Chat",
+			description = "Configures if duplicate game chat can be collapsed",
+			position = 9
```

Comment:
Your indents here are one level too far.
```suggestion
		keyName = "collapseGameChat",
		name = "Collapse Game Chat",
		description = "Configures if duplicate game chat can be collapsed",
		position = 9
```

Reply:
Also the description for the added configs should be written into more active statements, eg. "Collapse duplicate game chat messages into a single line"